gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinClusterConfigurationKeys.java

```diff
@@ -149,6 +149,9 @@ public class GobblinClusterConfigurationKeys {
   public static final long DEFAULT_HELIX_JOB_STOP_TIMEOUT_SECONDS = 10L;
   public static final String TASK_RUNNER_SUITE_BUILDER = GOBBLIN_CLUSTER_PREFIX + "taskRunnerSuite.builder";
 
+  public static final String HELIX_JOB_WAIT_COMPLETION_TIMEOUT_SECONDS = GOBBLIN_CLUSTER_PREFIX + "helix.job.waitCompletionTimeoutSeconds";
+  public static final long DEFAULT_HELIX_JOB_WAIT_COMPLETION_TIMEOUT_SECONDS = 60L;
```

Comment:
60 seconds is too less time for a job to complete

Reply:
increased to 300 seconds

Reply:
increase further to 15 mins